src/context_diff.rs

```diff
@@ -263,13 +265,16 @@ fn make_diff(
 }
 
 #[must_use]
+#[allow(clippy::too_many_arguments)]
```

Comment:
i wonder if this could not be refactored at some point

Reply:
Yes, it's on my to-do list, will do in a separate PR.